### DISCUSSION TITLE
Console client shutting when accessing a 1.18.2 Skyblock Server due to Resource pack

### DISCUSSION THREAD

**Discussion by Nurseyyy** [ORelio/Minecraft-Console-Client#2148] [Q&A]
Hi there, I have been trying to use the MCC to afk my alt on a server. The server forces a resource pack else you get kicked if you don't use it. The console client constantly closes because of this, but it closes so quickly once the message pops up  that I'm not able to get a screen shot of it. I'm unsure as to what to do to get around this. Does any one have any suggestions? Can supply the name of the server if needed :)

**Comment by BruceChenQAQ** [comment]
Probably the same reason as #2129.
Try using this [pre-release](https://github.com/BruceChenQAQ/Minecraft-Console-Client-1.19dev/releases/tag/bugfix_pre_build) and test if the problem still exists.

**Reply by Nurseyyy** [reply]
removed comment as just total reinstall of an earlier version fix it :)
